ui/StatusQ/src/StatusQ/Layout/StatusMainLayout.qml

```diff
@@ -55,8 +55,7 @@ SplitView {
     }
 
     Control {
-        SplitView.minimumWidth: 78
-        SplitView.preferredWidth: 78
+        SplitView.preferredWidth: !!leftPanel && leftPanel.visible ? leftPanel.width : 0
```

Comment:
This is to keep the previous behaviour of the shell being full screen. IMO, it looks fine when the tabbar is still shown when the Shell is visible, but we can easily revert.